share/spice/bible/bible.js

```diff
@@ -17,7 +17,8 @@
             meta: {
                 sourceName: 'Bible',
                 sourceUrl: source_url,
-                sourceIconUrl: 'http://bible.org/sites/bible.org/files/borg6_favicon.ico'
+                sourceIconUrl: 'http://bible.org/sites/bible.org/files/borg6_favicon.ico',
+                signal: 'high'
```

Comment:
Ah, you were right first time. This should be above the `meta` object, next to `id`, `name`, etc. Sorry!